InnerEye/ML/dataset/sample.py

```diff
@@ -154,19 +152,20 @@ class Sample(SampleBase):
     image: Union[np.ndarray, torch.Tensor]
     # (Batches if from data loader) x Z x Y x X
     mask: Union[np.ndarray, torch.Tensor]
-    # (Batches if from data loader) x Classes x Z X Y x X
-    labels: Union[np.ndarray, torch.Tensor]
+    # (Batches if from data loader) x Classes x Z X Y x X, where the first class is background
+    labels: Optional[Union[np.ndarray, torch.Tensor]]
     metadata: PatientMetadata
+    missing_labels: List[bool]
```

Comment:
I see that you have had to change a LOT of tests because this field is mandatory. You can make it optional by saying `missing_labels: Optional[List[bool]] = None`, and recognize missing list as meaning "everything is provided".